test/kerberos_tests.js

```diff
@@ -168,5 +168,24 @@ describe('Kerberos', function () {
         });
       });
     });
+
+    describe('options.user', function () {
+      context('valid values for `user`', function () {
+        test('no options provided', async function () {
+          const rs = await client.wrap('x'.repeat(100));
+          expect(rs).length.to.be.greaterThan(100);
```

Comment:
We noticed the bug because the wrapped value was actually shorter than the original challenge text. This is just a simple assertion to demonstrate that the result is plausible.

When we _are_ providing a user, the challenge text is still discarded and therefore the result is shorter.